packages/kernel/build.yaml

```diff
@@ -3,6 +3,16 @@ requires:
   category: "distro"
   version: ">=0"
 
+excludes:
+- ^/boot/grub2
+- ^/boot/initrd
+{{ if .Values.codename }}
+{{if eq .Values.codename "teal" }}
+unpack: true
+includes:
+- ^/boot|^/boot/.*
```

Comment:
The real difference in teal is that we extract all the kernels from the image, so no zypper in or build happening